qutebrowser/browser/commands.py

```diff
@@ -1847,13 +1663,23 @@ def debug_webaction(self, action, count=1):
             action: The action to execute, e.g. MoveToNextChar.
             count: How many times to repeat the action.
         """
-        member = getattr(QWebPage, action, None)
-        if not isinstance(member, QWebPage.WebAction):
+        tab = self._current_widget()
+
+        if tab.backend == tabmod.Backend.QtWebKit:
```

Comment:
I don't really get what you mean with "Otherwise you'd need to repeat this on every call-site and e.g. plugin authors may forget to do this." - on every call site of what?

The reason why there's this distinction here is because `debug_webaction` (which is only here for the command `:debug-webaction`) gets the action as a string, and I'd prefer having real enum values in the tab API. So the command handler has to convert the string to an enum value (while discarding invalid ones).

Or do you mean having an abstraction over all webactions so you could pass the same value to either? Then the problem is that they are too backend specific, and I'd rather have abstractions for specific goals (like `caret`). But then I still want/need `:debug-webaction` to trigger a specific action when playing around.